packages/RuntimeValueVisualization-Tools.package/.squot-contents

```diff
@@ -1,5 +1,6 @@
 SquotTrackedObjectMetadata {
 	#objectClassName : #PackageInfo,
+	#ignoredInstanceVariables : IdentityDictionary { },
```

Comment:
is this a new default in Squot or a local experiment?

Reply:
Well, [apparently neither nor](https://github.com/hpi-swa/Squot/issues/367), I'm removing it again. :-)

```suggestion
```